_sitegen/controllers/COSummary.js

```diff
@@ -103,6 +103,23 @@ function showConfirmation(order) {
                 dataObj = orderConfirmationData.getData(order, exchangeID);
                 serviceCallResult = klaviyoUtils.trackEvent(exchangeID, dataObj, klaviyoUtils.EVENT_NAMES.orderConfirmation, order.customerEmail);
             }
+
+            if ('KLEmailSubscribe' in session.custom || 'KLSmsSubscribe' in session.custom) {
+                var email = session.custom.KLEmailSubscribe ? order.customerEmail : false;
+                var phone = session.custom.KLSmsSubscribe ? order.defaultShipment.shippingAddress.phone : false;
+                var e164PhoneRegex = new RegExp(/^\+[1-9]\d{1,14}$/);
+                if (phone) {
+                    // NOTE: Klaviyo only accepts phone numbers that include + and the country code at the start (ex for US: +16465551212)
+                    // in order to successfully get users subscribed to SMS list you must collect the country code in your order phone number field!
+                    phone = '+' + phone.replace(/[^a-z0-9]/gi, '');
+                    if (!e164PhoneRegex.test(phone)) {
+                        phone = false;
```

Comment:
This will throw an error on the API request if we are passing it as false - maybe set to `null` here

Reply:
same as above